plugins/zenoh-plugin-rest/src/lib.rs

```diff
@@ -285,16 +288,8 @@ impl Plugin for RestPlugin {
         WORKER_THREAD_NUM.store(conf.work_thread_num, Ordering::SeqCst);
         MAX_BLOCK_THREAD_NUM.store(conf.max_block_thread_num, Ordering::SeqCst);
 
-        let task = run(runtime.clone(), conf.clone());
-        let task = blockon_runtime(async {
-            timeout(Duration::from_millis(1), TOKIO_RUNTIME.spawn(task)).await
-        });
+        spawn_runtime(run(runtime.clone(), conf.clone()));
```

Comment:
The original design is to stop zenohd if unable to run REST. I'm not sure whether we want to change the behavior here.